### ISSUE TITLE
onChangeText cause flicking when is set on textInputProps

### ISSUE THREAD

**Issue by volpejoaquin** [FaridSafi/react-native-google-places-autocomplete#632]
# Describe the bug

I'm using `GooglePlacesAutocomplete` component with `textInputProps.onChangeText` set. I think there is a conflict with the internal `onChangeText={_handleChangeText}` bind, this cause a flick on my app and doesn't work property

# Reproduction

Steps to reproduce the behavior - a minimal reproducible code example, link to a [snack](https://snack.expo.io) or a repository.

  ```javascript
  <GooglePlacesAutocomplete
       minLength={3}
       fetchDetails={true}
       onPress={handlePlaceSelect}
       renderDescription={renderDescription}
       query={PLACES_QUERY}
       textInputProps={{
          onChangeText: handleChange,
          returnKeyType: "done",
        }}
       ...
   /> 
  ```

# Additional context

- Library Version: 0.2.3
- React Native Version: 0.63.3

- [X] iOS
- [X] Android

# Error
![error](https://user-images.githubusercontent.com/1527191/95930253-5f4eef80-0d9c-11eb-8704-25f48a066d88.gif)

The problem is here:
https://github.com/FaridSafi/react-native-google-places-autocomplete/blob/master/GooglePlacesAutocomplete.js#L793

# Possible fix 1
Destructure `onChange` prop from `props.textInputProps`

```javascript
let {
    onFocus,
    onBlur,
    clearButtonMode,
    InputComp,
    onChangeText,
    ...userProps
  } = props.textInputProps;
```
[HERE](https://github.com/FaridSafi/react-native-google-places-autocomplete/blob/master/GooglePlacesAutocomplete.js#L748)

# Possible fix 2
Invert order of `{...userProps}  and `onChangeText={_handleChangeText}`

```javascript
  {...userProps}
  onChangeText={_handleChangeText}
```
[HERE](https://github.com/FaridSafi/react-native-google-places-autocomplete/blob/master/GooglePlacesAutocomplete.js#L793)

## Fixed

![error-fixed](https://user-images.githubusercontent.com/1527191/95930260-6544d080-0d9c-11eb-9986-8f7a198aff05.gif)

**Comment by bell-steven** [comment]
Thank for the detailed bug report. Would you mind putting in a PR?

**Comment by bell-steven** [comment]
closed by #647
